docs/modules/operation/pages/performance-data-collection/collectd/configuration.adoc

```diff
@@ -1,22 +1,22 @@
 
```

Comment:
Combine lines 26 and 27 into one line to match how it shows in the default collectd-configuration.xml